packages/table/src/Cell.tsx

```diff
@@ -5,30 +5,58 @@ import { commonCellStyles } from './styles';
 
 export const tdInnerDiv = createDataProp('td-inner-div');
 
-type CellProps = HTMLElementProps<'td', HTMLTableCellElement>;
+interface HeaderCellProps
+  extends HTMLElementProps<'th', HTMLTableHeaderCellElement> {
+  isHeader?: true;
```

Comment:
Don't think this should be optional. isHeader is necessarily true for us to be using HeaderCellProps

Reply:
You're right